README.md

```diff
@@ -1,15 +1,73 @@
-# GitHub Repo Template
+# Audio ADC - LabVIEW
+
+## Overview
+
+'Audio ADC - LabVIEW' is a MeasurementLink based plugin that enables users to perform measurements of Audio Analog to Digital Converters.
+
+The measurement plugin generate an analog audio signal input to the Device Under Test and measures the Digital Output.
+
+![image](res/audio-adc-plugin.png)
+
+
+## Key Features
+
+ - Generates analog audio signal supporting below protocols
+   - I2S
+   - Left Justified
+   - Right Justified
+   - TDM
+ 
+ - Single tone measurements
+   - THD
+   - THD+N
+   - SNR
+   - SFDR
+   - Dynamic range
+   - Gain error
+   - Output power
+
+
+## Hardware Setup
```

Comment:
We can add this line as user will understand SMU is not dependency **(note: the software does not use the SMU/power supply but need something to power the DUT)** I will keep it to u where to put it